website/docs/getting-started/tutorials/customer-support-tool/add-new-tickets.md

```diff
@@ -0,0 +1,180 @@
+    - Delete the default **Close** and **Confirm** buttons.
+    - Increase the width of the Modal using the resize handle.
+
+:::tip
+If you accidentally close the Modal, you can open it again by selecting the name of the modal **mdlNewTicket** under the **Explorer** tab in the **Widgets** section.
+:::
+
+2. From under the **Widgets** tab, drop a Form widget inside the Modal. Delete the default **Form** title. Increase the width of the Form using the resize handle.
+
+3. Drop two Input and three Select widgets inside the Form and set their properties as follows:
+
+    <b><u>Properties - Input1</u></b>
+
+    - **Widget Name**: `c_user` <br/>
+    - **Data type**: `Email`<br/>
+    - **Text**: `User Email`<br/>
+    - **Required**: `true`
+
+    <br/><b><u>Properties - Input2</u></b>
+
+    - **Widget Name**: `c_description` <br/>
+    - **Data type**: `Multi-line text`<br/>
+    - **Text**: `Description`<br/>
+    - **Required**: `true`
+
+    <br/><b><u>Properties - Select1</u></b>
+
+    - **Widget Name**: `c_category` <br/>
+    - **Options**: 
+
+    ```javascript
+    [
+      {
+        "label": "Billing",
+        "value": "Billing"
+      },
+      {
+        "label": "Account",
+        "value": "Account"
+      },
+      {
+        "label": "Technical",
+        "value": "Technical"
+      }
+    ]
+    ```
+
+    - **Default selected value**: empty<br/>
+    - **Label**: `Category`
+
+    <br/><b><u>Properties - Select2</u></b>
+
+    - **Widget Name**: `c_assignee` <br/>
+    - **Options**: 
+
+    ```javascript
+    [
+      {
+        "label": "John Doe",
+        "value": "John Doe"
+         },
+      {
+        "label": "Karmila Fox",
+        "value": "Karmila Fox"
+      },
+      {
+        "label": "Sarah Smith",
+        "value": "Sarah Smith"
+      }
+    ]
+    ```
+
+    - **Default selected value**: empty<br/>
+    - **Label**: `Assignee`
+
+    <br/><b><u>Properties - Select3</u></b>
+
+    - **Widget Name**: `c_priority` <br/>
+    - **Options**: 
+
+    ```javascript
+    [
+      {
+        "label": "High",
+        "value": "high"
+      },
+      {
+        "label": "Medium",
+        "value": "medium"
+      },
+      {
+        "label": "Low",
+        "value": "low"
+      }
+    ]
+    ```
+
+    - **Default selected value**: empty<br/>
+    - **Label**: `Priority`
+
+    The output should look something like this: 
+
+<figure>
+  <img src="/img/add-new-ticket-modal.png" style= {{width:"800px", height:"auto"}} alt="New Ticket Modal"/>
+  <figcaption align = "center"><i>Fig 1. New Ticket Modal</i></figcaption>
+</figure>
+
+## Create add ticket query
+
+You have to create an insert query to add the data entered in the Form into the database.
+
+1. Select the **Explorer** tab on the *Entity Explorer* to the screen's left. 
+
+2. Click the **+** icon next to **Queries/JS**. 
+
+3. Select **supportTickets query** from the list of options. 
```

Comment:
```suggestion
3. Select **supportTickets** query from the list of options. 
```

Reply:
this is correct. In the UI it shows up that way
<img width="556" alt="Screenshot 2023-06-28 at 11 14 23" src="https://github.com/appsmithorg/appsmith-docs/assets/32227135/b42d4d85-e169-4673-a453-78d4829bdf01">